src/gui_common/dialogs/TutorialDialog.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Godot;
+
+/// <summary>
+///   A window dialog for tutorials with custom behaviors such as show (pop-up) delay and animation.
+/// </summary>
+public class TutorialDialog : CustomDialog
+{
+    private Tween tween;
+
+    /// <summary>
+    ///   Tweakable delay to make tutorial sequences flow more naturally.
+    /// </summary>
+    [Export]
+    public float ShowDelay { get; set; } = 1.0f;
```

Comment:
I think this should be reduced to 0.7 because with this much delay I think the editor tutorials don't popup fast enough when going through the editor tabs.